specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```
@@ -6389,7 +6503,8 @@
         "kind": {
           "description": "The kind of the entity query",
           "enum": [
-            "Expansion"
+            "Expansion",
+            "Insights"
```

Here we define the enum value as `Insights` but below it is listed as `Insight` (as well as the discriminator values). Please fix this. The simplest way:
```suggestion
            "Insight"
```

---

Fixed. It should be Insight.